### PR TITLE
CRNS-32: Handling channel-truncated event

### PR DESCRIPTION
**Pull request by vishalnarkhede**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

## Description of the pull request

PR in react sdk - https://github.com/GetStream/stream-chat-react/pull/91
PR in RN sdk - https://github.com/GetStream/stream-chat-react-native/pull/92

### CODE REVIEW

File: test/presence.js [hunk 1]
```diff
@@ -104,7 +104,11 @@ describe('Presence', function() {
 					results.push([e.watcher_count, e.user.id]);
 					// expect to see thierry join, james join and james leave
 					if (results.length === 3) {
-						const expected = [[1, 'user1'], [2, 'james'], [1, 'james']];
+						const expected = [
+							[1, 'user1'],
+							[2, 'james'],
+							[1, 'james'],
+						];
```
Comment (ferhatelmas) [inline test/presence.js]: why do we have this diff?

Comment (vishalnarkhede) [reply test/presence.js]: Its a lint fix

Comment (vishalnarkhede) [reply test/presence.js]: Its a lint fix

Comment (ferhatelmas) [reply test/presence.js]: it's not - old version was fine. 
https://travis-ci.com/GetStream/stream-chat-js/jobs/261569670#L227

check and update your local prettier version.

**Pull request merged**